### DISCUSSION TITLE
Clickable/Selectable LineString

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2676] [Ideas]
Thanks to OM I have imported the Rights of Way data relating to Cumbria, England via a KML file.

This is a key piece of information relating to the access rights to Ways. In order to maintain these public ways individuals walk them to record defects and report back to the Authority who maintain them. So I added a Start, End and Middle Points so that the user could click on them and an information pane would give further information. It would be neat if the user could just select the Line rather than one of these Points/pins as it clutters up the screen. 
Can you just **select** a line
<img width="524" alt="Screenshot 2022-06-05 at 13 11 41" src="https://user-images.githubusercontent.com/58628197/172049989-d5b72856-27e4-4e14-9c6e-f8e737408cb3.png">
?

**Comment by biodranik** [comment]
It is possible if someone adds a code to select and highlight the line, together with iOS and Android code to display a different UI for that selection.

**Reply by pwb999** [reply]
Thanks. Is this KML code ?? And if so perhaps an example ?

**Reply by biodranik** [reply]
No, it's C++, Java and Swift/Objective C

**Reply by pwb999** [reply]
Ah...Much too hard for me alas. Many thanks for the help though. Perhaps we might see this as an option in a future release of OM.
Really using OM on the MacBook. Is there a Windows 11 version by chance as it would expose the benefits of OM to a wider community.

**Reply by biodranik** [reply]
It is possible to build a Windows version with some effort. The question is only about the importance and other priorities.

**Comment by pwb999** [comment]
I completely understand the difficulties in prioritising all the requests your team gets. Whilst I have used a Mac for some years I appreciate that the vast majority of computer users especially those in poorer countries ie Africa probably only have a basic PC to use or even a Linux machine. OSM derived maps are really useful in these countries and we should try to be as inclusive as possible. I understand that many users are car based and want navigation programs etc. However for the vast majority of users might just need to see a map of their area on whatever hardware they are lucky to have. Second its much easier to plan a long route and print it off on a Desktop machine. 
Anyway very many thanks for maintaining such a great app.
